solutions/standard/variables.tf

```diff
@@ -354,3 +354,15 @@ variable "elasticsearch_full_version" {
   type        = string
   default     = null
 }
+
+variable "image_port" {
+  description = "The port which is used to connect to the port that is exposed by the container image."
```

Comment:
Needs to be more specific here. You need to mention its related to kibana and only applicable if `enable_kibana_dashboard` is true

Reply:
Done